build-support/bin/_generate_all_lockfiles_helper.py

```diff
@@ -99,36 +100,37 @@ def jvm(cls, tool: type[JvmToolBase], *, backend: str | None = None) -> DefaultT
     DefaultTool.python(Autoflake),
     DefaultTool.python(Bandit, backend="pants.backend.python.lint.bandit"),
     DefaultTool.python(Black),
+    DefaultTool.python(ClangFormat, backend="pants.backend.experimental.cc.lint.clangformat"),
+    DefaultTool.python(CoverageSubsystem),
+    DefaultTool.python(DebugPy),
     DefaultTool.python(Docformatter),
+    DefaultTool.python(DockerfileParser, backend="pants.backend.docker"),
     DefaultTool.python(Flake8, source_plugins=True),
+    DefaultTool.python(IPython),
     DefaultTool.python(Isort),
-    DefaultTool.python(Pylint, backend="pants.backend.python.lint.pylint", source_plugins=True),
-    DefaultTool.python(Yapf, backend="pants.backend.python.lint.yapf"),
+    DefaultTool.python(Lambdex, backend="pants.backend.awslambda.python"),
+    DefaultTool.python(MyPy, source_plugins=True),
+    DefaultTool.python(PoetrySubsystem),
```

Comment:
This was the only real change (addition), the rest was alphabetizing the two groups (Python tools and JVM tools) individually.